changelogs/unreleased/4589-convert-a-v1-module-to-a-v2-with-the-presence-of-a-setuppy-file.yml

```
@@ -0,0 +1,6 @@
+---
+description: Reject v1tov2 module conversion when a setup.py is present
+change-type: patch
+destination-branches: [master, iso5, iso4]
```

The `v1tov2` command is not available on ISO4
```suggestion
destination-branches: [master, iso5]
```